### ISSUE TITLE
VB -> C#: Implicit conversions not performed

### ISSUE THREAD

**Issue by mrmonday** [icsharpcode/CodeConverter#267]
### Input code
```vbnet
Public Class Class1
    Sub Foo()
        Dim x As Decimal = 4
        Dim y As Double = 5
        Dim z = x + y
    End Sub
End Class
```

### Erroneous output
```c#
public class Class1
{
    public void Foo()
    {
        decimal x = 4;
        double y = 5;
        var z = x + y;
    }
}
```

### Expected output
```c#
public class Class1
{
    public void Foo()
    {
        decimal x = 4;
        double y = 5;
        var z = Convert.ToDouble(x) + y;
    }
}
```

### Details
Product in use: VS extension

Version in use: 6.6.0

VB performs some implicit conversions for arithmetic operations - these should be added explicitly when converting to C#.

The error from the C# is:
```
CS0019	Operator '+' cannot be applied to operands of type 'decimal' and 'double'
```

**Comment by mrmonday** [comment]
Lots of examples where this fails, eg. `""c = ""`.

There's a full list of these here:

* https://docs.microsoft.com/en-us/dotnet/visual-basic/reference/language-specification/expressions
* https://docs.microsoft.com/en-us/dotnet/visual-basic/reference/language-specification/conversions

**Comment by GrahamTheCoder** [comment]
This was fixed and tested in #300
